src/server.js

```diff
@@ -13,8 +14,7 @@ app.get("/", (req, res) => {
 });
 
 // API routes
-// Routes
-app.use("/api/v1", authRoutes);
+app.use("/api/v1/", router);
```

Comment:
I think this server file will be cleaner this way. We import all routes into the routes/index.js and then export the root route from there into this line.